app/assets/stylesheets/components/_btn.scss

```diff
@@ -111,3 +111,30 @@
     text-decoration: none;
   }
 }
+
+.usa-button.usa-button--unstyled:visited {
+  // Temporary: Links in the IdP do not currently conform to the design system and instead retain
+  // their color even if visited. Part of the work of LG-3877 should be to remove these styles, and
+  // instead allow both unstyled buttons and links inherit the default design system visited color.
+  // Alternatively, consider removing unstyled button classes from links, since the intention of an
+  // unstyled button is to take the visual appearance of a link.
+  color: $link-color;
+}
+
+.usa-button.usa-button--unstyled:hover,
+.usa-button.usa-button--hover.usa-button--unstyled {
+  // Temporary: Links in the IdP do not currently conform to the design system and instead retain
+  // their color while hovered. Part of the work of LG-3877 should be to remove these styles, and
+  // instead allow both unstyled buttons and links inherit the default design system hover color.
+  color: $link-color;
+}
+
+.usa-button--unstyled {
+  &:hover,
+  &:active {
+    // Temporary: These styles should be ported upstream to the design system, optionally as part of
+    // future reconciliation effort with uswds/uswds#4077.
+    -moz-osx-font-smoothing: inherit;
+    -webkit-font-smoothing: inherit;
+  }
+}
```

Comment:
This is becoming a bit of an embarrassing game of whack-a-mole, since this was already tried to be fixed across two separate patch releases in https://github.com/18F/identity-style-guide/pull/201 and https://github.com/18F/identity-style-guide/pull/202 🤦 

The problem now is that the first set of changes in https://github.com/18F/identity-style-guide/pull/201 reset font smoothing for hover and active states, but notably only for the explicit `--hover` and `--active` class modifiers that USWDS doesn't handle (https://github.com/uswds/uswds/pull/4077). The native `:hover` and `:active` states are inherited from USWDS, which applies its opinionated font smoothing ([source](https://github.com/uswds/uswds/blob/1908d1391bc59410624ca1934cc70b7404e8f443/src/stylesheets/core/mixins/_button-unstyled.scss#L13-L15)), and takes precedent over the base styles that were addressed in https://github.com/18F/identity-style-guide/pull/202.

For future follow-up:

- We could do _yet another_ patch release
- Or, wait to reconcile this 'til if / when https://github.com/uswds/uswds/pull/4077 is merged and published
- Or, wait to reconcile this 'til if / when https://github.com/uswds/uswds/pull/4115 is merged and published
- Or, propose an upstream change to USWDS to avoid the potentially-redundant [hover/active state font smoothing styles](https://github.com/uswds/uswds/blob/1908d1391bc59410624ca1934cc70b7404e8f443/src/stylesheets/core/mixins/_button-unstyled.scss#L15), which would further simplify wrangling various states

Reply:
> For future follow-up:

See: #206